src/components/UserPortal/ChatRoom/ChatHeader.tsx

```
@@ -0,0 +1,38 @@
+    <div className={styles.header}>
+      <div className={styles.userInfo}>
+        <ProfileAvatarDisplay
+          imageUrl={chatImage}
+          fallbackName={chatTitle}
+          className={styles.contactImage}
+          enableEnlarge={true}
+        />
+        <div
+          onClick={() => (isGroup && onGroupClick ? onGroupClick() : null)}
+          className={styles.userDetails}
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)